RAMP/packer.py

```diff
@@ -3,10 +3,25 @@
 import zipfile
 import click
 import yaml
+import semantic_version
 
 import RAMP.module_metadata as module_metadata
 from RAMP.commands_discovery import discover_modules_commands
 
+def version_to_semantic_version(version):
+    """
+    Creates a semantic version from a numeric redis module version.
+    """
+
+    major = int(version / 10000)
+    version -= major * 10000
+    minor = int(version / 100)
+    version -= minor * 100
+    patch = int(version)
+
+    sem_version_str = '{}.{}.{}'.format(major, minor, patch)
```

Comment:
format them with %02d so that they will be lexically sortable as well (i.e. 10.10.10 > 02.01.03)

Reply:
For the timebeing i'm leaving this one out as python semantic_version package complains with  "Result ValueError(u"Invalid leading zero in major: '03.02.01'",)"
We'll attend to this later on, feel free to open an issue.

Reply:
oh goddammit this is going to be a clusterfuck